cmd/proxy/main.go

```diff
@@ -63,19 +70,30 @@ func buildCLIOptions() *cli.App {
 	return app
 }
 
-func startProfile() {
+func startProfile(c *cli.Context) {
+	port := 6060
+	if c.IsSet(config.PProfPortFlag) { // Allow for port=0 to select a random port.
+		port = c.Int(config.PProfPortFlag)
+	}
+	if port < 0 {
+		return // pprof server disabled
```

Comment:
maybe we only enable profiler if port is provided? `-1` seems less intuitive.

Reply:
I just thought we'd want to enable it by default